src/App.js

```diff
@@ -1,16 +1,32 @@
-import Button from "./components/Button";
-
+import React, { useRef } from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+import Home from "./pages/Home";
+import Catalogue from "./pages/Catalogue";
+import CurtainBuilder from "./pages/CurtainBuilder";
+import Consultation from "./pages/Consultation";
+import ShoppingCart from "./pages/ShoppingCart";
```

Comment:
Organise imports somting like  (libs -> components -> utils -> mocks -> assets) for better readability of code